index.html

```diff
@@ -81,6 +81,235 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
+                                Back End Dev
+                            </li>
+                            <li class="Job-title_and_year">
+                                2022
+                            </li>
+                        </ul>
+                    </section>
+                    <p class="notice font-style">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="programming-languages font-style">
+                        <li>html</li>
+                        <li class="on-large">Ruby On Rails</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button id= "3" class="project-button button" type="button">See Project</button>
+                </section>
+            </section>
+        </div>
+        <div id="about" class="about-me">
+            <section class="written-info">
+                <section>
+                    <h2 class="about-me-heading font-style">About <br> Myself</h2>
+                    <p class="description font-style">
+                        I’m a software developer! I can help you build a product , feature or
+                        website Look through some of my work and experience! If you like what
+                        you see and have a project you need coded, don’t hestiate to contact
+                        me.
+                    </p>
+                </section>
+
+                <section>
+
+                    <h4 class="connect font-style">LET'S CONNECT</h4>
+                    <ul class="socials">
+                        <li>
+                            <a class="social_medias twitter" href="https://twitter.com/KNurgi">
+                                <img src="images/Vector (1).svg" alt="twitter logo">
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="social_medias" href="https://www.linkedin.com/in/kenbon-teshome/">
+                                <svg xmlns="http://www.w3.org/2000/svg" width="23" height="23" fill="currentColor"
+                                    class="bi bi-linkedin icon" viewBox="0 0 16 16">
+                                    <path
+                                        d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z" />
+                                </svg>
```

Comment:
- [ ] Kindly use the correct icon for LinkedIn like the Figma design, you can export it from Figma and use it 

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/212112534-551d7cd0-2a3a-4e47-abb0-71596f600579.png)   |   ![image](https://user-images.githubusercontent.com/83360636/212112639-09f943e4-1d94-44db-9236-0c497ca17bff.png)    |